presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java

```diff
@@ -96,7 +96,7 @@
     private HiveCompressionCodec hiveCompressionCodec = HiveCompressionCodec.GZIP;
     private boolean respectTableFormat = true;
     private boolean immutablePartitions;
-    private boolean createEmptyBucketFiles = true;
+    private boolean createEmptyBucketFiles;
```

Comment:
do we have tests for empty buckets with and without empty files?

Reply:
I don't know how not to create empty files in Hive on MR.

Reply:
https://hadoop.apache.org/docs/r3.1.1/hadoop-project-dist/hadoop-common/FileSystemShell.html#touchz ?

Reply:
Or using webfs that is provided in tempto

Reply:
we have good test coverage -- `TestHive` is run with hive 1.2 (includes empty files) and hive 3.1 (skips empty files).